05_deploy_bootstrap_vm.sh

```diff
@@ -124,15 +124,15 @@ sudo systemctl reload NetworkManager
 cat /tmp/dnsmasq.conf | ssh -o "StrictHostKeyChecking=no" core@$IP sudo dd of=dnsmasq.conf
 
 # Build and start the ironic container
-cat ironic/runironic.sh | ssh -o "StrictHostKeyChecking=no" core@$IP sudo dd of=runironic.sh
-cat ironic/Dockerfile | ssh -o "StrictHostKeyChecking=no" core@$IP sudo dd of=Dockerfile
-ssh -o "StrictHostKeyChecking=no" core@$IP sudo podman build \
-    --build-arg RHCOS_IMAGE_URL=${RHCOS_IMAGE_URL} \
-    --build-arg RHCOS_IMAGE_VERSION=${RHCOS_IMAGE_VERSION} \
-    --build-arg RHCOS_IMAGE_FILENAME_OPENSTACK=${RHCOS_IMAGE_FILENAME_OPENSTACK} \
-    -t ironic:latest .
+cat "${RHCOS_IMAGE_FILENAME_OPENSTACK}.gz" | ssh -o "StrictHostKeyChecking=no" "core@$IP" sudo dd of="${RHCOS_IMAGE_FILENAME_OPENSTACK}.gz"
+ssh -o "StrictHostKeyChecking=no" "core@$IP" sudo podman pull docker.io/vrutkovs/metalkube-ironic
 ssh -o "StrictHostKeyChecking=no" core@$IP sudo podman run \
-    -d --net host --privileged --name ironic -v /run:/run:shared -v /dev:/dev localhost/ironic
+    -d --net host --privileged --name ironic \
+    -v /home/core/dnsmasq.conf:/etc/dnsmasq.conf \
+    -v "/home/core/${RHCOS_IMAGE_FILENAME_OPENSTACK}.gz:/var/www/html/images/${RHCOS_IMAGE_FILENAME_OPENSTACK}" \
+    -v /run:/run:shared \
+    -v /dev:/dev \
```

Comment:
We no longer need these bind mounts
https://github.com/metalkube/dev-scripts/pull/48

Reply:
Thanks, fixed